docs/contributing/README.md

```diff
@@ -1,9 +1,142 @@
+efficient contributor workflow.
 
-In this section:
+Knative is a diverse, open, and inclusive community.
 
-- [Contribute to Knative](contributing.md): helpful tips for how to get started contributing to Knative.
-- [About the Knative community](about.md): information about the Knative community and how it's run.
+Knative is designed for different personas:
+
+![Diagram that displays different Audiences for Knative](images/knative-audience.svg)
+
+Knative components offer developers Kubernetes-native APIs for deploying
+serverless-style functions, applications, and containers to an auto-scaling
+runtime.
+
+To join the conversation, head over to the
+[Knative users](https://groups.google.com/d/forum/knative-users) Google group.
+
+Knative components are intended to be integrated into more polished products
+that cloud service providers or in-house teams in large enterprises can then
+operate.
+
+Any enterprise or cloud provider can adopt Knative components into their own
+systems and pass the benefits along to their customers.
+
+## Community Meetup
+
+This virtual event is designed for end users, a space for our community to meet,
+get to know each other, and learn about uses and applications of Knative.
+
+Catch up with past community meetups on our [YouTube channel](https://www.youtube.com/playlist?list=PLQjzPfIiEQLLyCyLBKLlwDLfE_A-P7nyg).
+
+Stay tuned for new events by subscribing to the
+[calendar](https://calendar.google.com/calendar/embed?src=knative.team_9q83bg07qs5b9rrslp5jor4l6s%40group.calendar.google.com&ctz=America%2FLos_Angeles) ([iCal export file](https://calendar.google.com/calendar/ical/knative.team_9q83bg07qs5b9rrslp5jor4l6s%40group.calendar.google.com/public/basic.ics))
+and following us on [Twitter](https://twitter.com/KnativeProject).
+
+## More ways to get involved
+
+Even if there’s not an issue opened for it, we can always use more testing throughout the platform. Similarly, we can always use more docs, richer docs, insightful docs. Or maybe a cool blog post? And if you’re a web developer, we could use your help in spiffing up our public-facing web site.
+
+[Bug reports](https://github.com/knative/serving/issues/new) and friction logs
+from new developers are especially welcome.
+
+## Knative authors
+
+For a current list of the authors, see [Authors](https://github.com/knative/serving/blob/main/AUTHORS).
+
+## Getting started
+
+Before you can make a contribution to Knative, you must sign the CNCF [EasyCLA](https://easycla.lfx.linuxfoundation.org/) using the same email address you used to register for Github.
+
+For more information, see [Contributor license agreements](https://github.com/knative/community/blob/main/CONTRIBUTING.md#contributor-license-agreements).
+
+Read the [Knative contributor guide](https://github.com/knative/community/blob/main/CONTRIBUTING.md).
+
+Read the [Code of conduct](https://github.com/knative/community/blob/main/CODE-OF-CONDUCT.md).
+
+## Governance
```

Comment:
Since governance is basically a giant link farm that's mostly of interest to people who want to know the nitty-gritty of how the project operates, I moved it from `about.md` to `governance.md`, but kept it as a separate document.